### PR TITLE
Add rocket fuel level widget for FDR

### PR DESCRIPTION
**Pull request by maxslarsson**

### CODE REVIEW

**Comment by maxslarsson** [comment]
* **#163** <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/163?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/163?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Review by ZachGarcia42 — COMMENTED** [review]
You should do FDR instead of this...

File: telemetry-proxy/telemetry_proxy
Comment (ZachGarcia42) [inline telemetry-proxy/telemetry_proxy:1]: Why is this changed?

**Review by ZachGarcia42 — CHANGES_REQUESTED** [review]
The length of the green bar changes based on the width of the widget

**Comment by ZachGarcia42** [comment]
> The length of the green bar changes based on the width of the widget

Wait it's random nevermind

**Review by ZachGarcia42 — APPROVED** [review]
Looks great! Super cool Widget!

**Comment by maxslarsson** [comment]
### Merge activity

* **Nov 16, 9:22 PM EST**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/163).

**Pull request merged**